www/src/components/site-metadata.js

```diff
@@ -21,28 +21,43 @@ const SiteMetadata = ({ pathname }) => {
     }
   `)
 
-  return (
-    <Helmet defaultTitle={title} titleTemplate={`%s | ${title}`}>
-      <html lang="en" />
-      <link rel="canonical" href={`${siteUrl}${pathname}`} />
-      <meta name="docsearch:version" content="2.0" />
-      <meta
-        name="viewport"
-        content="width=device-width,initial-scale=1,shrink-to-fit=no,viewport-fit=cover"
-      />
-
-      <meta property="og:url" content={siteUrl} />
-      <meta property="og:type" content="website" />
-      <meta property="og:locale" content="en" />
-      <meta property="og:site_name" content={title} />
-      <meta property="og:image" content={`${siteUrl}${gatsbyIcon}`} />
-      <meta property="og:image:width" content="512" />
-      <meta property="og:image:height" content="512" />
+  const {
+    metaImage = `${siteUrl}${gatsbyIcon}`,
```

Comment:
This should maybe (?) prepend siteUrl so that we can just pass the absolute path as metaImage and always prefix with the siteUrl.

Reply:
Gotcha, yea that would make sense to just always prefix `siteUrl` so that you don't have to hardcode `https://www.gatsbyjs.org` for every image